### PR TITLE
web: change Sidebar tab semantics

### PR DESCRIPTION
**Pull request by nicks**
Hello @hyu, @landism,

Please review the following commits I made in branch nicks/ch10949:

0415af82f97e65b92e8933b47337f9b472c3fdd4 (2021-01-25 11:50:45 -0500)
web: change Sidebar tab semantics
now, we never change the number of tabs open if the selected resource is already open

Code review reminders, by giving a LGTM you attest that:

* Commits are adequately tested
* Code is easy to understand and conforms to style guides
* Incomplete code is marked with TODOs
* Code is suitably instrumented with logging and metrics

### CODE REVIEW

File: web/src/TabNav.tsx [hunk 1]
```diff
@@ -86,26 +86,15 @@ export function LegacyNavProvider(
...
 //
-// 1. When you single click a resource on the left sidebar,
+// 1) When you click or ctrl-click a resource on the left sidebar that's already open,
 //    it changes the current tab to the new resource.
 //
-// 2. When you ctrl-click a resource on the left sidebar,
-//    it opens a new tab, and brings the new tab into focus.
+// 2) Otherwise, when you select a resource on the left sidebar,
+//    a) click opens it in the current tab,
+//    b) ctrl-click opens it in a new tab on the immediate right of current tab
```
Comment (hyu) [inline web/src/TabNav.tsx:94]: Not part of this PR, but I noticed during review:

For me on Firefox, the Mac OS context menu still has precedence. I assume the expected behavior is that Tilt should hijack?

(Maybe I'm missing something. You're able to override the context menu on Mac?)

**Review by hyu — APPROVED** [review]

File: web/src/TabNav.tsx (hunk 1 above)
Comment (nicks) [reply web/src/TabNav.tsx:94]: yep, on macos it's the command key

fwiw, most of the standards docs on this tend to be super vague and hand-wavey for exactly the reason you describe - this tends to be dictated by a complicated set of overlapping OS-level preferences and browser-level preferences, so i changed this comment to be more hand-wavey

**Review by landism — APPROVED** [review]
so many fewer cases!
